### PR TITLE
E0d/change name coursediscover

### PR DESCRIPTION
**Pull request by e0d**

### CODE REVIEW

**Comment by e0d** [comment]
@arbabnazar see this additional commit.

**Comment by e0d** [comment]
@zubair-arbi OK to merge this?

File: util/jenkins/ansible-provision.sh [hunk 1]
```diff
@@ -267,6 +267,11 @@ CREDENTIALS_SOCIAL_AUTH_REDIRECT_IS_HTTPS: true
 COURSE_DISCOVERY_OAUTH_URL_ROOT: "https://${deploy_host}"
 COURSE_DISCOVERY_URL_ROOT: "https://course-discovery-${deploy_host}"
 COURSE_DISCOVERY_SOCIAL_AUTH_REDIRECT_IS_HTTPS: true
+=======
+DISCOVERY_OAUTH_URL_ROOT: "https://${deploy_host}"
+DISCOVERY_URL_ROOT: "https://discovery-${deploy_host}"
+DISCOVERY_SOCIAL_AUTH_REDIRECT_IS_HTTPS: true
+>>>>>>> 5f930fc... modified the name inside all playbooks
```
Comment (cpennington) [inline util/jenkins/ansible-provision.sh]: Merge conflict

File: playbooks/roles/discovery/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+DISCOVERY_MEMCACHE: [ 'memcache' ]
+
+DISCOVERY_CACHES:
+  default:
+    BACKEND:  'django.core.cache.backends.memcached.MemcachedCache'
+    KEY_PREFIX: 'default'
+    LOCATION: '{{ DISCOVERY_MEMCACHE }}'
+
+DISCOVERY_VERSION: "master"
+DISCOVERY_DJANGO_SETTINGS_MODULE: "discovery.settings.production"
```
Comment (clintonb) [inline playbooks/roles/discovery/defaults/main.yml]: course_discovery.settings.production

**Comment by fredsmith** [comment]
:+1:

**Pull request merged**